source/_patterns/03-uswds/header/nav/nav.twig

```diff
@@ -0,0 +1,13 @@
+<nav class="usa-nav" role="navigation" aria-label="{{ label }}">
+  {% if is_extended %}<div class="usa-nav__inner">{% endif %}
+    <button class="usa-nav__close">{{ 'Close'|t }}</button>
+    {% block content %}
+      {% include "@uswds/header/nav/_primary-nav.twig" with {
+        "prefix": is_extended ? "extended" : "basic"
+      } %}
+      {% if is_extended %}
+        {% include "@uswds/header/secondary/secondary.twig" %}
+      {% endif %}
+    {% endblock %}
```

Comment:
Minor nitpick: this is indented one level too deep

Reply:
My eyes are failing me. Which line(s) are too far indented?

Reply:
Lines 3-11